robottelo/ui/locators.py

```diff
@@ -1844,6 +1844,12 @@ def __iter__(self):
         ("//input[@placeholder='Filter' and contains(@ng-model, 'Search']")),
     "gpgkey.product_repo": (
         By.XPATH, "//td/a[contains(@href, 'repositories')]"),
+    "gpgkey.success_upload_alert": (
+        By.XPATH,
+        "//div[@ng-hide]//div[contains(@class, 'alert-success')]"),
+    "gpgkey.error_upload_alert": (
```

Comment:
Xpath is pretty common here. Are you sure it's gpgkey-specific only? I wonder maybe the same alerts were added somewhere else so it's better to move them to `common_locators`?